### ISSUE TITLE
Package still refers to and contains old workflow

### ISSUE THREAD

**Issue by mdingemanse** [elpaco-escience/talkr#112]
The packagedown pages somehow still contain the old workflow: https://elpaco-escience.github.io/talkr/articles/workflow.html

The new one is also there: https://elpaco-escience.github.io/talkr/articles/workflows.html

The new one should replace the old one everywhere

**Comment by bvreede** [comment]
Good catch! This happens sometimes; the quick fix is to re-run the website build manually. The [workflow that does this](https://github.com/elpaco-escience/talkr/actions/workflows/pkgdown.yaml) has a manual trigger:
![Screenshot 2024-08-23 at 11 04 13](https://github.com/user-attachments/assets/802191dd-c280-4808-acd1-75f4d05978c2)

Pressing the 'run workflow' button and running it on the `main` branch will recreate the page and deploy it to GitHub Pages. It takes a few minutes to run and deploy, but should fix this problem if it occurs again. I just did this, and it has [updated the page](https://elpaco-escience.github.io/talkr/index.html) — refresh if it doesn't work!

**Comment by mdingemanse** [comment]
Actually it's still the case that description refers to https://elpaco-escience.github.io/talkr/articles/workflow.html (singular) which is the old one that doesn't reflect my work in the user-journeys branch. The right URL to the vignette is https://elpaco-escience.github.io/talkr/articles/workflows.html (plural). I can correct that (just did in https://github.com/elpaco-escience/talkr/compare/main...112-fix-old-workflow-html) but I still wonder why that old one even still exists.

Root cause is my renaming `workflow.Rmd` to `workflows.Rmd` but that change was clearly merged to main (workflow.Rmd (singular) doesn't exist anymore). So, I'm confused.

**Comment by bvreede** [comment]
Very possible. @PabRod do you know this?

I'll check the PR!

**Comment by mdingemanse** [comment]
Ready to be closed as the PR is merged, but leaving open to solve the issue of an old `workflow.html` hanging around somewhere

**Comment by PabRod** [comment]
I'll take a look at this, but let's tag it as low priority for now.
